openrpc.json

```diff
@@ -766,10 +767,20 @@
             "type": "object",
             "properties": {
               "snapId": {
-                "description": "The ID of the Snap to invoke."
+                "description": "The ID of the Snap to invoke.",
+                "type": "string"
               },
               "request": {
-                "description": "The JSON-RPC request object to send to the invoked Snap."
+                "description": "The JSON-RPC request object to send to the invoked Snap.",
+                "type": "object",
+                "properties": {
+                  "method": {
+                    "type": "string"
+                  },
+                  "params": {
+                    "type": "string"
```

Comment:
should this be left as 'any' ? I guess our interactive docs dont really know what to do with it in that case though...